spec/lib/aliquot/payment_spec.rb

```diff
@@ -0,0 +1,160 @@
+require 'aliquot-pay'
+
+require 'json'
+require 'openssl'
+
+describe Aliquot::Payment do
+  let(:token) { AliquotPay.generate_token_ecv1(@payment, key, recipient) }
+  let(:token_string) { JSON.unparse(token) }
+  let(:shared_secret) { extract_shared_secret(token, recipient) }
+  let(:cipher) { OpenSSL::Cipher::AES128.new(:CTR) }
+
+  it 'decrypts successfully' do
+    @payment = AliquotPay.payment
```

Comment:
Why is this an instance variable, and why is it never used?

Reply:
We've been through this IRL, it will most likely be changed in a separate branch.